fake-news-detection/src/fake_news_detection/pipelines/data_processing/__init__.py

```
@@ -0,0 +1 @@
+#from .pipeline import create_pipeline, onion_pipeline
```

Ruff (0.11.9)

No newline at end of file

Add trailing newline

(W292)